internal/fs/schemas/builder.json

```diff
@@ -236,16 +233,15 @@
             },
             "commands": {
               "description": "Additional CLI commands to add",
-              "$ref": "#/definitions/commands",
-              "minLength": 1
```

Comment:
ah, I think we can make this `minItems` then

https://json-schema.org/understanding-json-schema/reference/array.html#length